src/gui/windows/debug.zig

```diff
@@ -61,6 +62,16 @@ pub fn render() void {
 		y += 8;
 		draw.print("Queue size: {}", .{main.threadPool.queueSize()}, 0, y, 8, .left);
 		y += 8;
+		if (main.frameCount % main.perfUpdateFrequency == 0)
+			main.lastPerformance = main.threadPool.performance.readAndReset();
```

Comment:
Is there any reason not to just use the moving average and never resetting?
I think with a moving average it should be fine to update the value every frame even.

Reply:
Are we certain performance will not vary over time? I can imagine different biomes, especially colorado, being significantly different.

Reply:
Hmm, that is probably true.
The reason I prefer not having it reset is because averaging over 10000s of requests can yield more comparable performance numbers.

So maybe a manual reset would be best. Could either add a separate button, or just tie it to the debug window visibility.